src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -111,6 +111,7 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings)
         {
             // Take the passed in sources
             var packageSources = new HashSet<string>(Sources, StringComparer.Ordinal);
+            var sourceObjects = new Dictionary<string, PackageSource>(StringComparer.Ordinal);
```

Comment:
That's what I would've put by default, but the previous line and the Distinct() call later in this function both use Ordinal. So I guess in this XPlat code the Uris are treated as case sensitive...

---
In reply to: [62935539](https://github.com/NuGet/NuGet.Client/pull/586#discussion_r62935539) [](ancestors = 62935539)